### DISCUSSION TITLE
Dịch passthrough neuron

### DISCUSSION THREAD

**Discussion by minhduc0711** [mlbvn/handson-ml2-vn#105] [Terminology] [answered]
Context:
> The inputs of the Perceptron are fed to special **passthrough neurons** called input neurons: they output whatever input they are fed. 

@cuongvng đang đề xuất là `nơ-ron xuyên`

cc @mlbvn/handson-ml

**Comment by hnguyentt** [comment]
Mình đề xuất:
* `nơ-ron xuyên suốt`
* `nơ-ron xuyên qua`
* `nơ-ron xuyên thấu`

**Reply by minhduc0711** [reply]
Em thấy `xuyên qua` có vẻ ok nhất

**Comment by rootonchair** [comment]
Từ này khá lạ, có ai biết nguồn nào khác định nghĩa từ này không? Em search không thấy, giống như tác giả tự tạo thêm từ này vậy

**Reply by minhduc0711** [reply]
Đúng là từ này không thấy ai dùng thật..

**Reply by hnguyentt** [reply]
Mà thấy nó cũng hơi Ko có chức năng gì vì input vào sao thì output ra i chang vậy.

**Comment by ngcthuong** [comment, marked as answer]
Giông như kiểu "chuyển tiếp" hơn.
